### PR TITLE
[tldr] Adjust spacing of tldr block

### PR DESCRIPTION
**Pull request by SondraE**
## Description
I thought the spacing looked a little strange in the tl;dr section of the PWA comparison articles. I put TL;DR on the same line as the rest of the text in that block.

## Steps to test/reproduce
/2023/07/11/when-to-choose-a-responsive-web-app/
and the other 2 articles in the series

### CODE REVIEW

File: content/blog/2023/when-to-choose-a-native-mobile-app.md [hunk 1]
```diff
@@ -44,8 +44,7 @@ App](/2023/07/11/when-to-choose-a-responsive-web-app/)
 
 ## Native Mobile App -- Best for Advanced Capabilities
 
-> **tl;dr**
->
+> **TL;DR**
```
Comment (jgerigmeyer) [inline content/blog/2023/when-to-choose-a-native-mobile-app.md:47]:
```suggestion
> **TL;DR** --
```

File: content/blog/2023/when-to-choose-a-progressive-web-app.md [hunk 2]
```diff
@@ -48,8 +48,7 @@ comparison chart for quick reference).
 
 ## Progressive Web App -- Best of Both Worlds
 
-> **tl;dr**
->
+> **TL;DR**
```
Comment (jgerigmeyer) [inline content/blog/2023/when-to-choose-a-progressive-web-app.md:51]:
```suggestion
> **TL;DR** --
```

File: content/blog/2023/when-to-choose-a-responsive-web-app.md [hunk 3]
```diff
@@ -73,8 +73,7 @@ Here are the biggest differences in a nutshell.
 
 ## Responsive Web Apps -- Best for Wide Reach
 
-> **tl;dr**
->
+> **TL;DR**
```
Comment (jgerigmeyer) [inline content/blog/2023/when-to-choose-a-responsive-web-app.md:76]:
```suggestion
> **TL;DR** --
```

**Review by jgerigmeyer — COMMENTED** [review]
@SondraE I find this a bit harder to read -- what do you think about adding a dash if you prefer them on the same line?

**Review by jgerigmeyer — APPROVED** [review]
